### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by Azahires**

### CODE REVIEW

**Review by SOA-P — DISMISSED** [review]

File: frontend/src/components/DisplayOffers/DisplayOffers.jsx [hunk 1]
```diff
@@ -8,8 +8,14 @@ import { Context } from "../../contexts/Context";
 export default function DisplayOffers() {
   const [offers, setOffers] = useState([]);
   const { searchValue } = useContext(Context);
-  const api = useApi();
-  useEffect(() => {
+  const { filterCdd } = useContext(Context);
+  const { filterCdi } = useContext(Context);
+  const { filterAlternance } = useContext(Context);
+  const { filterInternship } = useContext(Context);
```
Comment (LoicBrassart) [inline frontend/src/components/DisplayOffers/DisplayOffers.jsx]: const { searchValue,filterCdd,filterCdi ,filterAlternance,filterInternship} = useContext(Context);

File: frontend/src/components/FilterPanel/FilterPanel.jsx [hunk 2]
```diff
@@ -1,5 +1,76 @@
...
+import { useContext } from "react";
+import { Context } from "../../contexts/Context";
 import Style from "./style";
 
 export default function FilterPanel() {
-  return <Style>FilterPanel</Style>;
+  const { filterCdd, setFilterCdd } = useContext(Context);
+  const { filterCdi, setFilterCdi } = useContext(Context);
+  const { filterAlternance, setFilterAlternance } = useContext(Context);
+  const { filterInternship, setFilterInternship } = useContext(Context);
```
Comment (LoicBrassart) [inline frontend/src/components/FilterPanel/FilterPanel.jsx]: same

File: frontend/src/components/FilterPanel/FilterPanel.jsx [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
+
+          {/* <select name="disatnce-max" id="selector">
+            <option value="">Sélectionne ta distance maximale</option>
+            <option value="5">5 Km</option>
+            <option value="10">10 Km</option>
+            <option value="25">25 Km</option>
+            <option value="50">50 Km</option>
+            <option value="100">100 Km</option>
+            <option value="all">Indifférent</option>
+          </select> */}
```
Comment (LoicBrassart) [inline frontend/src/components/FilterPanel/FilterPanel.jsx]: dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
some evils, plz fix

**Review by Azahires — COMMENTED** [review]
Fixing all evil !

**Review by SOA-P — APPROVED** [review]

**Review by SOA-P — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**